uuid.go

```diff
@@ -68,7 +68,7 @@ const (
 )
 
 // Timestamp is the count of 100-nanosecond intervals since 00:00:00.00,
-// 15 October 1582 within a V1 UUID. This type has no meaning for V2-V5
+// 15 October 1582 within a V1 UUID. This type has no meaning for V3-V5
```

Comment:
I would simplify this a bit:

```suggestion
// 15 October 1582 within a V1 UUID. This type has no meaning for other
```